src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -1107,58 +1124,7 @@ function getCategories(
     return [];
 }
 
-/**
- * Transform
- *      viewByTwoAttributes: {
- *          parent: [P1, P1, P2, P2, P3],
- *          children: [C1, C2, C1, C2, C2]
- *      }
- * to
- *      [{
- *          name: P1,
- *          categories: [C1, C2]
- *       }, {
- *          name: P2,
- *          categories: [C1, C2]
- *       }, {
- *          name: P3,
- *          categories: [C2]
- *       }]
- * @param viewByTwoAttributes
- */
-export function getCategoriesForTwoAttributes(viewByTwoAttributes: IViewByTwoAttributes): Array<{
-    name: string,
-    categories: string[]
-}> {
-    const { parent = [], children = [] } = viewByTwoAttributes;
-    const combinedResult = parent.reduce((
-        result: { [property: string]: string[] },
-        parentAttr: Execution.IResultAttributeHeaderItem,
-        index: number
-    ) => {
-        const key: string = get(parentAttr, 'attributeHeaderItem.name', '');
-        const value: string = get(children[index], 'attributeHeaderItem.name', '');
-
-        const childCategories: string[] = result[key] || [];
-
-        return {
-            ...result,
-            [key]: [...childCategories, value] // append value
-        };
-    }, {});
-
-    return Object.keys(combinedResult).map((name: string) => ({
-        name,
-        categories: combinedResult[name]
-    }));
-}
-
-/**
- * Get stacking config which will be set to 'highchart.plotOptions.series'
- * @param stackByAttribute
- * @param options
- */
-function getStackingConfig(stackByAttribute: IUnwrappedAttributeHeadersWithItems, options: IChartConfig): string {
+function getStackingConfig(stackByAttribute: any, options: IChartConfig): string {
```

Comment:
why did you replace IUnwrappedAttributeHeadersWithItems with any?

Reply:
Fixed. It's my bad.